From 5e62fd4f1c15e510b8261688e0f7a8152fbe0e19 Mon Sep 17 00:00:00 2001
From: Tony Cheng <tony.cheng@amd.com>
Date: Thu, 25 Aug 2016 23:22:32 -0400
Subject: [PATCH 0943/1722] drm/amd/dal: remove get_encoder_number in bios.

DC does not use this to construct link, as DC look up
encoder from connector.src

Change-Id: I8062393c1a235d6d5aae3a45e2e7e983ddae0d75
Signed-off-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c | 2 --
 drivers/gpu/drm/amd/dal/dc/dc_bios_types.h    | 1 -
 2 files changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
index e1bf8c3..a71acbf 100644
--- a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
+++ b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
@@ -4364,8 +4364,6 @@ static void bios_parser_destroy_integrated_info(
 static const struct dc_vbios_funcs vbios_funcs = {
         .get_connectors_number = bios_parser_get_connectors_number,
 
-        .get_encoders_number = bios_parser_get_encoders_number,
-
         .get_oem_ddc_lines_number = bios_parser_get_oem_ddc_lines_number,
 
         .get_encoder_id = bios_parser_get_encoder_id,
diff --git a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
index 2bc868d..e928419 100644
--- a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
+++ b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
@@ -40,7 +40,6 @@
 
 struct dc_vbios_funcs {
 	uint8_t (*get_connectors_number)(struct dc_bios *bios);
-	uint8_t (*get_encoders_number)(struct dc_bios *bios);
 	uint32_t (*get_oem_ddc_lines_number)(struct dc_bios *bios);
 
 	struct graphics_object_id (*get_encoder_id)(
-- 
2.7.4

